core/audits/metrics/first-meaningful-paint.js

```diff
@@ -1,89 +0,0 @@
-import {FirstMeaningfulPaint as ComputedFmp} from '../../computed/metrics/first-meaningful-paint.js';
-
-const UIStrings = {
-  /** Description of the First Meaningful Paint (FMP) metric, which marks the time at which a majority of the content has been painted by the browser. This is displayed within a tooltip when the user hovers on the metric name to see more. No character length limits. The last sentence starting with 'Learn' becomes link text to additional documentation. */
-  description: 'First Meaningful Paint measures when the primary content of a page is ' +
-      'visible. [Learn more about the First Meaningful Paint metric](https://developer.chrome.com/docs/lighthouse/performance/first-meaningful-paint/).',
-};
-
-const str_ = i18n.createIcuMessageFn(import.meta.url, UIStrings);
-
-class FirstMeaningfulPaint extends Audit {
-  /**
-   * @return {LH.Audit.Meta}
-   */
-  static get meta() {
-    return {
-      id: 'first-meaningful-paint',
-      title: str_(i18n.UIStrings.firstMeaningfulPaintMetric),
-      description: str_(UIStrings.description),
-      scoreDisplayMode: Audit.SCORING_MODES.NUMERIC,
-      supportedModes: ['navigation'],
-      requiredArtifacts: ['traces', 'devtoolsLogs', 'GatherContext', 'URL'],
-    };
-  }
-
-  /**
-   * @return {{mobile: {scoring: LH.Audit.ScoreOptions}, desktop: {scoring: LH.Audit.ScoreOptions}}}
-   */
-  static get defaultOptions() {
-    return {
-      mobile: {
-        // 25th and 5th percentiles HTTPArchive -> median and PODR, then p10 derived from them.
-        // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2018_04_01_mobile?pli=1
-        // see https://www.desmos.com/calculator/i4znkdccut
-        scoring: {
-          p10: 2336,
-          median: 4000,
-        },
-      },
-      desktop: {
-        // SELECT QUANTILES(renderStart, 21) FROM [httparchive:summary_pages.2018_12_15_desktop] LIMIT 1000
-        scoring: {
-          p10: 934,
-          median: 1600,
-        },
-      },
-    };
-  }
-
-  /**
-   * Audits the page to give a score for First Meaningful Paint.
-   * @see https://github.com/GoogleChrome/lighthouse/issues/26
-   * @see https://docs.google.com/document/d/1BR94tJdZLsin5poeet0XoTW60M0SjvOJQttKT-JK8HI/view
-   * @param {LH.Artifacts} artifacts The artifacts from the gather phase.
-   * @param {LH.Audit.Context} context
-   * @return {Promise<LH.Audit.Product>}
-   */
-  static async audit(artifacts, context) {
-    const trace = artifacts.traces[Audit.DEFAULT_PASS];
-    const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
-    const gatherContext = artifacts.GatherContext;
-    const metricComputationData = {trace, devtoolsLog, gatherContext,
-      settings: context.settings, URL: artifacts.URL};
-    const metricResult = await ComputedFmp.request(metricComputationData, context);
-    const options = context.options[context.settings.formFactor];
-
-
-    return {
-      score: Audit.computeLogNormalScore(
```

Comment:
Would it be possible to keep around an audit shell on the off chance someone is still using this? Removing the entire implementation but just prevent users from seeing a `Cannot read properties of undefined` error?

Thinking it would just return `score: null, notApplicable: true` or whatever. Obviously remove completely once the next breaking change hits.